### PR TITLE
website/docs: clarify troubleshooting headlines and order

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

-   **Does this resolve an issue?**
    Resolves #

### New Features

-   Adds feature which does x, y, and z.

### Breaking Changes

-   Adds breaking change which causes \<issue\>.

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/troubleshooting/login.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Troubleshooting Login problems
+title: I can't login to authentik
```
Comment (tanberry) [inline website/docs/troubleshooting/login.md:2]:
```suggestion
title: I can't log in to authentik
```

File: website/sidebars.js [hunk 2]
```diff
@@ -302,12 +302,12 @@ module.exports = {
                     },
                 },
                 "troubleshooting/access",
-                "troubleshooting/csrf",
-                "troubleshooting/emails",
                 "troubleshooting/login",
                 "troubleshooting/image_upload",
                 "troubleshooting/missing_permission",
                 "troubleshooting/missing_admin_group",
+                "troubleshooting/csrf",
```
Comment (smusali) [inline website/sidebars.js:309]: all looks good - I just wonder why the positions of these have to change.

**Review by tanberry — APPROVED** [review]

File: website/sidebars.js (hunk 2 above)
Comment (BeryJu) [reply website/sidebars.js:309]: image
with the new titles the new order makes more sense imo

**Review by smusali — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**